### ISSUE TITLE
Window switcher. Minimizing a single open application and then immediately doing alt+tab breaks the switcher and the session.

### ISSUE THREAD

**Issue by marco4mp** [elementary/gala#1343]
### What Happened?

When opening a single application and then minimizing and quickly doing alt + tab weird things happen:
The name of the app in the switcher changes to "n/a", then the switcher is not responding, then the switcher appears in the upper left corner. Finally if you try something else like opening the application menu or the multitasking view everything goes to black and the login screen appears. 

### Steps to Reproduce

https://user-images.githubusercontent.com/52709674/146614276-df2e7699-ea77-4cf0-bf20-70c3ee43d3d2.mp4

(Obviously I had to stop and save the recording before the fifth step).

1. Open a single app
2. Minimize the application (I use ctrl + h)
3. Immediately press alt + tab.   --> The switcher shows "n/a" instead of the app name.
4. Press alt + tab again.             --> The switcher appears in the upper left corner.
5. Select multitasking view.        --> Black screen, the session is closed and the login screen is displayed.

### Expected Behavior

It should work normally

### OS Version

6.x (Odin)

### Software Version

Latest release (I have run all updates)

### Hardware Info

Intel® Core™ i5-7200U CPU @ 2.50GHz
Intel® HD Graphics 620 (Kaby Lake GT2)
NVIDIA GM108M [GeForce 920MX]

**Comment by TomiOhl** [comment]
Partially can confirm, I accidentally run into this numerous times a day, but it doesn't happen every time and for me only with chrome browser.

**Comment by TomiOhl** [comment]
Hmm, maybe commit 26ebdae58a6c42ce9a1cd60d12608ab9a1918389 could fix this one too?

**Comment by TomiOhl** [comment]
Seems to be fixed in Gala 6.3.1.

**Comment by marco4mp** [comment]
I can't check it right now because of this new issue https://github.com/elementary/gala/issues/1401   😕

**Comment by RobinBoers** [comment]
I have run into this bug a few times before. When trying to reproduce it today something strange happened: I opened Music in a new workspace, minimized it using ctrl+h, and then tried to switch to it using alt+tab. It worked, but there was a weird black border around the app and the top panel disappeared. After a second I got a black screen and got switched back to the first workspace. When I tried again it showed the Music icon, but with the title of Firefox and it switched me back to the first workspace where Firefox was.

Here's a video of the problem: https://www.youtube.com/watch?v=ells_tTJz_Q
